tests/dummy/app/controllers/index.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Controller.extend({
 
   allCookies: computed(function() {
     let cookieService = this.get('cookies');
-    cookieService.write('now', new Date().getTime());
+    cookieService.write('now', new Date().getTime(), { domain: 'localhost' });
```

Comment:
We do not need. I just added it to make sure we support that. If you don't have the fix for host, this example would break. That's why I added this.